src/test/ui/consts/const-eval/const_fn_ptr_fail2.stderr

```diff
@@ -1,22 +1,14 @@
-error[E0080]: evaluation of constant expression failed
-  --> $DIR/const_fn_ptr_fail2.rs:20:5
+error[E0080]: evaluation of constant value failed
+  --> $DIR/const_fn_ptr_fail2.rs:20:16
    |
 LL |     assert_eq!(Y, 4);
-   |     ^^^^^^^^^^^-^^^^^
```

Comment:
Previously it used `tcx.promoted_mir_opt_const_arg(def.to_global())[p].span` for the span of a promoted, now it uses `ecx.cur_span()`... at least I think that's what is causing the error output changes here. The new spans are better than the old ones though, in my opinion.